graphium/finetuning/finetuning.py

```diff
@@ -0,0 +1,72 @@
+from typing import Iterable, List, Dict, Tuple, Union, Callable, Any, Optional, Type
+
+import pytorch_lightning as pl
+
+from torch.optim.optimizer import Optimizer
+from pytorch_lightning.callbacks import BaseFinetuning
+
+
+class GraphFinetuning(BaseFinetuning):
+    def __init__(self, cfg, train_bn: bool = False):
```

Comment:
Change to explicitly pass parameters. 
```suggestion
    def __init__(self, fine-tuning, architecture, module_from_pretrained, ....................., train_bn: bool = False):
```

Reply:
Addressed in 76e2ba6777935056030eefb9f82dfc0b53a3884d